pom.xml

```diff
@@ -304,7 +304,7 @@
         <dependency>
             <groupId>org.eclipse.jdt</groupId>
             <artifactId>org.eclipse.jdt.annotation</artifactId>
-            <version>1.1.400</version>
+            <version>2.2.300</version>
```

Comment:
This is a major upgrade and causes issues with type annotations because Skript is __not__ written in Java 8 in mind. It's written in Java 6/7 and updated to Java 8 afterwards. This possible with tons of hassle but it does not worth it since the 1.x version is also supported.